### PR TITLE
fix for codemirror width

### PR DESCRIPTION
**Pull request by pree-T**
#11696  a possible fix for this. Kindly review

### CODE REVIEW

File: packages/cells/style/inputarea.css [hunk 1]
```diff
@@ -59,6 +62,7 @@
   .jp-InputArea-editor {
     display: table-row;
     margin-left: var(--jp-notebook-padding);
+
```
Comment (fcollonval) [inline packages/cells/style/inputarea.css:65]: Unneeded empty line

```suggestion
```

File: packages/cells/style/inputarea.css [hunk 2]
```diff
@@ -40,6 +42,7 @@
   font-size: var(--jp-code-font-size);
   border: var(--jp-border-width) solid transparent;
   opacity: var(--jp-cell-prompt-opacity);
+  width: var(--jp-cell-prompt-width);
```
Comment (fcollonval) [inline packages/cells/style/inputarea.css:45]: Could you remove the `min-width` rule?

File: packages/cells/style/inputarea.css [hunk 3]
```diff
@@ -11,11 +11,13 @@
...
   display: table;
   overflow: hidden;
+  table-layout: fixed;
+  width: 100%;
 }
 
 .jp-InputArea-editor {
   display: table-cell;
-  width: 100%;
+
```
Comment (fcollonval) [inline packages/cells/style/inputarea.css:20]: Unneeded empty line

```suggestion
```

File: tsconfigdoc.json [hunk 4]
```diff
@@ -20,274 +20,274 @@
   },
   "references": [
     {
-      "path": "./packages/application"
+      "path": "./packages\\application"
```
Comment (fcollonval) [inline tsconfigdoc.json:23]: Would you mind reverting the change in this file please?

**Comment by fcollonval** [comment]
Open question: this changes the width of the input prompt to a constant width. Is this acceptable? Do we need to apply the same change in the output area (I would say yes)?

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @pree-T I made some suggestions. And they are one question regarding the need to apply similar changes to the output area that is likely to be confirmed by others.

**Comment by fcollonval** [comment]
I looked again at the problematic PR https://github.com/jupyterlab/jupyterlab/pull/11508/files

Actually the error comes from the style `flex: 0 0 <width>` that have been replaced by a `min-width` constrain when they should have been replaced by `width` constraint.

So could you please replace `min-width` by `width` in both:

https://github.com/jupyterlab/jupyterlab/blob/4a9c1d52c760bde51e0af0ec60a46ac3053cb78c/packages/outputarea/style/base.css#L31

https://github.com/jupyterlab/jupyterlab/blob/4a9c1d52c760bde51e0af0ec60a46ac3053cb78c/packages/cells/style/widget.css#L96

And set that table as `table-layout: fixed;` as you did for the input area.

https://github.com/jupyterlab/jupyterlab/blob/4a9c1d52c760bde51e0af0ec60a46ac3053cb78c/packages/outputarea/style/base.css#L22

**Comment by pree-T** [comment]
Sure Sir, I'll make all the necessary changes. Thanks for the guidance .

**Pull request closed without merging**